usecases/config/environment.go

```diff
@@ -479,9 +480,9 @@ func FromEnv(config *Config) error {
 		config.EnableApiBasedModules = true
 	}
 
-	config.AutoSchema.Enabled = true
+	config.AutoSchema.Enabled = runtime.NewDynamicValue(true)
 	if v := os.Getenv("AUTOSCHEMA_ENABLED"); v != "" {
-		config.AutoSchema.Enabled = !(strings.ToLower(v) == "false")
+		config.AutoSchema.Enabled.SetDefault(!(strings.ToLower(v) == "false"))
```

Comment:
nit: `!entcfg.Enabled(...)` instead of  `!(strings.ToLower(v) == "false")`

Reply:
I will skip this, because it's irrelevant to my changes. Plus here they check for exactly "false" whereas `entcfg.Enabled()` does bit differently. 

I just don't want to change the behavior on this PR.